frontend/src/shared/Interfaces.ts

```diff
@@ -17,6 +17,8 @@ export interface User {
   id: string;
   username: string;
   avatarUrl: string;
+  displayName: string;
+  techs: Array<String>;
```

Comment:
If we are populating the techs field in the backend to be an array of ITech (e.g. `techs: ITech[]`), we will want to make sure that matches the frontend interface here too